tests/unit/source_code/test_functional.py

```diff
@@ -244,7 +244,6 @@ def test_functional(sample: Functional, mock_path_lookup, simple_dependency_reso
         ("_child_that_uses_missing_value.py", "parent_that_dbutils_runs_child_that_misses_value_from_parent.py"),
         ("_child_that_uses_value_from_parent.py", "grand_parent_that_dbutils_runs_parent_that_magic_runs_child.py"),
         ("_child_that_uses_missing_value.py", "parent_that_imports_child_that_misses_value_from_parent.py"),
-        ("_child_that_uses_value_from_parent.py", "grand_parent_that_imports_parent_that_magic_runs_child.py"),
```

Comment:
This test is removed as it tests a non-existing situation: a file imports a notebook that runs another notebook. It does not work because the imported notebook is considered to be a file because our import resolver always returns files and not notebooks (as it should)